src/test/query.test.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { expect } from 'chai';
+
+describe('GraphQL Hello Query', () => {
+  it('should "Hello World"', async () => {
```

Comment:
acho que tem que ajustar a descrição, talvez um return?
```suggestion
  it('should return "Hello World"', async () => {
```